### PR TITLE
Use 'emit --info', 'emit --verbose', ...

### PR DESCRIPTION
**Pull request by floitsch**

### CODE REVIEW

File: src/ui.toit [hunk 1]
```diff
@@ -340,19 +448,83 @@ class Ui:
         if message:
           printer_.emit --kind=kind "$text.call"
 
+  /** Variant of $(emit --kind [--structured] [--text]) using the $DEBUG kind. */
+  emit --debug/bool [--structured] [--text]:
```
Comment (kasperl) [inline src/ui.toit:452]: Maybe all of these should have `-> none`.

File: src/ui.toit [hunk 2]
```diff
@@ -290,37 +302,133 @@ class Ui:
...
+  /** Variant of $(emit-map --kind map) using the $ERROR kind. */
+  emit-map --error/bool map/Map --title/string?=null:
+    emit-map --kind=ERROR map --title=title
+
+  /**
+  Variant of $(emit-map --kind map) using the $RESULT kind.
+
+  A program should do only a single result output per run.
+  */
+  emit-map --result/bool map/Map --title/string?=null:
```
Comment (kasperl) [inline src/ui.toit:429]: Usually we check that `result` is true in these cases.

**Review by kasperl — APPROVED** [review]
Looks good to me with two widely applicable comments.

File: src/ui.toit (hunk 2 above)
Comment (floitsch) [reply src/ui.toit]: done.

File: src/ui.toit (hunk 1 above)
Comment (floitsch) [reply src/ui.toit:503]: done.

**Pull request merged**